email-config-shell/src/requirements.txt

```diff
@@ -0,0 +1,4 @@
+cloudshell-shell-core>=5.0.3,<6.0.0
+cloudshell-email
+cloudshell-automation-api
+cloudshell-core
```

Comment:
please pin the dependencies to specific versions with a range like in line 1

Reply:
Pinned to fixed versions